fideslog/sdk/python/client.py

```diff
@@ -1,6 +1,8 @@
 # pylint: disable=too-many-arguments
 
-from asyncio import run
+import asyncio
+import sys
+from asyncio import run, set_event_loop_policy
```

Comment:
`WindowsSelectorEventLoopPolicy` is not importable on macOS, so we'll need to import it after verifying that the process is running on Windows. Also, let's keep these as organized and minimal as possible.
```suggestion
from asyncio import run, set_event_loop_policy
from sys import platform, version_info
```